### PR TITLE
#62 Added cli tool to export data in a csv format

### PR DESCRIPTION
**Pull request by klangner**
This tool is quite useful for me. In the futuree I plan to add support for selecting the columns and saving as json

### CODE REVIEW

**Review by WenyXu — APPROVED** [review]
LGTM

File: src/bin/orc-export.rs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+use clap::Parser;
+use orc_rust::ArrowReaderBuilder;
+
+#[derive(Parser)]
+#[command(name = "orc-export")]
+#[command(version, about = "Export data from orc file to csv", long_about = None)]
+struct Cli {
+    /// Path to the orc file
+    file: PathBuf,
+    /// Output file
```
Comment (Jefffrey) [inline src/bin/orc-export.rs:14]: Maybe need to enhance doc here to indicate it outputs to stdout if not provided?

File: src/bin/orc-export.rs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    // TODO: convert_dates
+    // TODO: format=[csv|json]
+    // TODO: columns="col1,col2"
+}
+
+fn main() -> Result<()> {
+    let cli = Cli::parse();
+    let f = File::open(&cli.file)?;
+    let output_writer: Box<dyn io::Write> = if let Some(output) = cli.output {
+        Box::new(File::create(output).unwrap())
```
Comment (Jefffrey) [inline src/bin/orc-export.rs:27]:
```suggestion
        Box::new(File::create(output)?)
```

Let's take advantage of the Result return type :+1:

File: src/bin/orc-export.rs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    } else {
+        Box::new(io::stdout())
+    };
+
+    let reader = ArrowReaderBuilder::try_new(f).unwrap().build();
```
Comment (Jefffrey) [inline src/bin/orc-export.rs:32]:
```suggestion
    let reader = ArrowReaderBuilder::try_new(f)?.build();
```

Ditto

Comment (klangner) [reply src/bin/orc-export.rs]: fixed

File: src/bin/orc-export.rs (hunk 2 above)
Comment (klangner) [reply src/bin/orc-export.rs]: fixed

File: src/bin/orc-export.rs (hunk 1 above)
Comment (klangner) [reply src/bin/orc-export.rs]: fixed

**Review by Jefffrey — APPROVED** [review]
Thanks for this :+1:

**Pull request merged**